global/fsl/etc/matlab/read_avw.m

```diff
@@ -24,6 +24,8 @@
 [status,output]=call_fsl(command);
 
 if (status),
+  delete([tmpname,'.hdr']);
+  delete([tmpname,'.img']);
```

Comment:
Is this addition just bringing our copy of `read_avw.m` into alignment with the current FSL version?  If not, maybe we should include a comment regarding our specific addition?

Reply:
Same comment applies to the other changes to the FSL matlab tools.

Reply:
These two lines are present in fsl 6.0.5.1.